course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -988,6 +988,38 @@ def _add_course_instructors(self, course_instructors, course_run):
             if not course_run.staff.filter(uuid=instructor.uuid).exists():
                 course_run.staff.add(instructor)
 
+    def _add_course_categories(self, categories, course_run):
+        """
+        Create and add categories to a course run.
+        """
+        course_run.course.subjects.clear()
+        for category in categories:
+            subject, created = Subject.objects.get_or_create(
+                marketing_id=category['id'],
+                partner=self.partner,
+                defaults={
```

Comment:
this can be simplified to `**category` if data keys are kept the same on both wordpress and discovery side.

Reply:
Keys are not same.